packages/playground/windows/playground/MainPage.xaml

```diff
@@ -168,6 +168,23 @@
                     <ComboBoxItem Content="V8" x:Name="x_engineV8" />
                 </ComboBox.Items>
             </ComboBox>
+            <TextBlock
+                Margin="4"
+                VerticalAlignment="Center"
+                Text="Theme:"/>
+            <ComboBox
```

Comment:
nit, could use ComboBox's Header property instead of a TextBlock

Reply:
I was following existing ComboBoxes in this file. Any idea why those don't use Header?

Reply:
we probably didn't think about it too much :)